src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$color-main-accent: rgb(6, 11, 53);
+$color-second: rgb(97, 96, 112);
```

Comment:
In my opinion it is a good practice to name color variable so it actualy saying what color it is like **color-black-main**